tests/graph/test_basic.py

```diff
@@ -806,3 +808,74 @@ def test_NominalVariable_create_variable_type():
     assert type(ntv_unpkld) is type(ntv)
     assert ntv_unpkld.equals(ntv)
     assert ntv_unpkld is ntv
+
+
+def test_is_dependent_on():
+    x = MyVariable(1)
+    x.name = "x"
+    y = MyVariable(1)
+    y.name = "y"
+    x2 = MyOp(x)
+    x2.name = "x2"
+    y2 = MyOp(y)
+    y2.name = "y2"
+    o = MyOp(x2, y)
+    assert is_dependent_on(o, [x])
+    assert is_dependent_on(o, [y])
+    assert not is_dependent_on(o, [y2])
+    assert not is_dependent_on(y, [y2])
+    assert is_dependent_on(y, [y])
+
+
+def test_variable_conditioning():
```

Comment:
Or whatever we end up calling the function. I agree with @aseyboldt that `condition_subset` is a very opaque name.

```suggestion
def test_condition_subset():
```